src/View/Helper/MigrationHelper.php

```diff
@@ -382,7 +382,7 @@ public function getColumnOption(array $options)
         if (isset($columnOptions['signed']) && $columnOptions['signed'] === true) {
             unset($columnOptions['signed']);
         }
-        if (empty($columnOptions['precision'])) {
+        if (is_null($columnOptions['precision'])) {
```

Comment:
Can we use !== instead?

Reply:
Sure I will change it.. I am dealing now with two issues in workflow. It is still using PHP 7.2 for code standards even when I have update it to 7.4 . I fill fix the workflow and then make this change.